tests/topotests/grpc_basic/test_basic_grpc.py

```diff
@@ -149,7 +149,8 @@ def test_get_config(tgen):
         }
       }
     ]
-  }
+  },
+  "frr-logging:logging": "*"
```

Comment:
I think @mjstapp point is to use `exact=False` when comparing the result so that no matter what YANG backward compat changes are added to modules in the future this test will keep passing.

Reply:
Ah, yep I can do that as well.